src/liboslexec/llvm_util.cpp

```diff
@@ -1386,7 +1389,6 @@ LLVM_Util::make_jit_execengine (std::string *err,
     options.RelaxELFRelocations = false;
     //options.DebuggerTuning = llvm::DebuggerKind::GDB;
 
-    options.PrintMachineCode = dumpasm();
```

Comment:
Just because its not in 12, doesn't mean it should be available for people building with LLVM 7, 8, 9, 10, or 11.  Could you wrap this with some conditional compilation to exclude it for LLVM version 12?  Better yet, is there a way to achieve same behavior with 12 (dumping assembly of jit'd code).  If not sure, maybe exclude it for 12 with a TODO comment to renable... somehow.

Reply:
Good catch, Alex!

Reply:
Will look into the if statement shouldn't be too hard to add again not mainly a c++ dev. Yes the same functionality is possible with other debug options of LLVM from what i read but i am pretty unfamiliar with it. Will see if i can find a solution. can't promise much be will add a todo if not.

Reply:
Thanks @EuanFH